community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -102,6 +102,10 @@
             "allows an upgrade to be performed.")
     public static final Setting<Boolean> allow_store_upgrade = setting("dbms.allow_format_migration", BOOLEAN, FALSE );
 
+
+    @Description( "Specifies custom storage format name. If not set default format will be used." )
+    public static final Setting<String> record_format = setting( "dbms.record_format", Settings.STRING, "" );
```

Comment:
This property should be used only if you want to override default format selection. Default format still edition specific.
Do you think it's make sense to describe it in description or change a name to something like: `dbms.record_format.override`?